crates/re_types/definitions/rerun/datatypes/uvec2d.fbs

```diff
@@ -0,0 +1,21 @@
+include "arrow/attributes.fbs";
+include "fbs/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+namespace rerun.datatypes;
+
+// ---
+
+/// A uint32 vector in 2D space.
+struct UVec2D (
+  "attr.arrow.transparent",
+  "attr.python.aliases": "npt.NDArray[Any], npt.ArrayLike, Sequence[int]",
+  "attr.python.array_aliases": "npt.NDArray[Any], npt.ArrayLike, Sequence[Sequence[int]], Sequence[int]",
+  "attr.rust.derive": "Default, Copy, PartialEq, Eq, PartialOrd, Ord, bytemuck::Pod, bytemuck::Zeroable",
```

Comment:
Again with `Ord`. Maybe it's fine, but maybe it's weird. If you are planning on puttitng them in a set, then `Hash` makes a lot more sense